cheerio-scraper/src/crawler_setup.js

```diff
@@ -267,7 +256,6 @@ class CrawlerSetup {
                 globalStore: this.globalStore,
                 requestQueue: this.requestQueue,
                 customData: this.input.customData,
-                useRequestQueue: this.input.useRequestQueue !== false,
             },
             pageFunctionArguments: {
```

Comment:
Can you elaborate a bit?

Reply:
Bad selection of lines to display. I was talking about the `pageFunctionArguments`. [More info about `CrawlingContext`](https://github.com/apify/apify-js/blob/master/MIGRATIONS.md#handler-arguments-are-now-crawling-context).

Basically, it would be the easiest for us to document and the easiest for users to understand, if the `pageFunction(context)` was exactly the same as `handlePageFunction(crawlingContext)` in `CheerioCrawler`.

I don't even remember why we only exposed some of the original arguments. Maybe to keep them "in sync" between Web, Puppeteer and Cheerio Scrapers. But I think we learned now that people expect Cheerio Scraper to behave exactly the same as CheerioCrawler and so on.

Reply:
So it should be now ok I guess? As the whole `crawlingContext` is deconstructed into the `pageFunctionArguments`. I kept the `response` override, not sure why it is done that way, maybe also to get around some cycles (or to mitigate differences as some scrapers have `res.status()` as a method rather than value).

Reply:
Yeah, exactly. It's because the `Response` objects are different across the crawlers.